code/modules/mob/living/simple_animal/hostile/regalrat.dm

```diff
@@ -30,16 +30,17 @@
 	ventcrawler = VENTCRAWLER_ALWAYS
 	unique_name = TRUE
 	faction = list("rat")
+	var/rummaging = FALSE
```

Comment:
add DMdoc to this by adding a comment like so above this line

`///This var does X`